RecoEgamma/EgammaPhotonProducers/python/mustacheOOTPhotonCore_cff.py

```diff
@@ -0,0 +1,8 @@
+import FWCore.ParameterSet.Config as cms
+from RecoEgamma.EgammaPhotonProducers.photonCore_cfi import *
+
+mustacheOOTPhotonCore = photonCore.clone()
+mustacheOOTPhotonCore.scHybridBarrelProducer = cms.InputTag("particleFlowSuperClusterOOTECAL:particleFlowSuperClusterOOTECALBarrel")
+mustacheOOTPhotonCore.scIslandEndcapProducer = cms.InputTag("particleFlowSuperClusterOOTECAL:particleFlowSuperClusterOOTECALEndcapWithPreshower")
+mustacheOOTPhotonCore.conversionProducer = cms.InputTag("conversions")
```

Comment:
If you would prefer to drop it, and just name this as ootPhoton*, I can do that.  There are no other plans to add any other OOT photon collection(s).